src/clr-angular/popover/signpost/signpost-content.ts

```diff
@@ -67,10 +67,12 @@ export class ClrSignpostContent extends AbstractPopover {
   /**********
    *
    * @description
-   * Close function that uses the signpost instance to toggle the state of the content popover.
+   * Close function that uses the IfOpenService instance to toggle the state of the content popover.
    *
    */
-  close() {
+  close(event: MouseEvent) {
+    event.preventDefault();
+    event.stopPropagation();
```

Comment:
`event.stopPropagation()` is a hard no in a library. The reason is that it completely breaks any listener that is added somewhere higher in the tree, whether it's been added by us or by the application.

Here are a couple examples:
- Consumers listening to clicks on the container of `event.stopPropagation()` element to trigger some action will never get the event. Typically this is done for performance reasons, adding one listener to a list of 1000 items instead of adding 1000 listeners.
- Elements that appear on top of the page, and close when you click anywhere. They will not close if you click on the element that has `event.stopPropagation()`.
- Global action trackers for analytics or business insights. They work by adding global `click`/`mousemove` listeners on the whole document, and `event.stopPropagation()` creates "dark zones" without anyone knowing.